docs/UserGuide.md

```diff
@@ -1,178 +1,145 @@
-
-### Archiving data files `[coming in v2.0]`
-
-_{explain the feature here}_
-
---------------------------------------------------------------------------------------------------------------------
-
-## FAQ
-
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
-
---------------------------------------------------------------------------------------------------------------------
-
-## Command summary
-
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+# User Guide
+NUSave is a **desktop app for managing expenditures, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, NUSave can help you manage and track expenses faster than traditional GUI apps.  
+  
+## Table of Contents  
+- [Quick Start](#quick-start)   
+- [Features](#features)  
+    * [Creating a budget book: `create`](#creating-a-budget-book-create)
+    * [Opening a budget book: `open`](#opening-a-budget-book-open)
+    * [Closing a budget book: `close`](#closing-a-budget-book-close)
+    * [Listing all budget books or expenditures: `list`](#listing-all-budget-books-or-expenditures-list)
+    * [Locating expenditures by name: `find`](#locating-expenditures-by-name-find)
+    * [Deleting a budget book or expenditure: `del`](#deleting-a-budget-book-or-expenditure-del)
+    * [Listing available commands: `help`](#listing-available-commands-help)
+    * [Exiting the program: `exit`](#exiting-the-program-exit)
+- [Command Summary](#command-summary)  
+  
+  
+## Quick Start  
+  
+1. Ensure you have Java `11` or above installed in your Computer.  
+      
+2. Download the latest `NUSave.jar` from [here](https://github.com/AY2021S1-CS2103T-T11-4/tp/releases).  
+      
+3. Copy the file to the folder you want to use as the *home folder* for your NUSave.  
+      
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds.  
+      
+5. Type the command in the command box and press enter to execute it. Some example commands you can try:  
+    - `list`: Lists all budget books.  
+      
+   - `create Temasek Hall`: Adds a budget book named Temasek Hall to NUSave.  
```

Comment:
Perhaps "Temasek Hall" is too specific to NUS? Users who are not from NUS may not understand what it means. "Daily Expenses" might be a better example.